figures/metrics.py

```diff
@@ -436,6 +436,28 @@ def calc_from_user_model():
     return calc_from_user_model()
 
 
+def get_total_site_staff_users_for_time_period(site, start_date, end_date):
```

Comment:
where is `start_date` being used?

Reply:
The helper method `get_monthly_history_metric` of figures used for calculating metrics has `start_data` and `end_date` required and to keep in line with that we set `start_date`. The `figures` do the same in methods where `start_date` or any other parameter is not used.

https://github.com/edly-io/figures/blob/ebe055db933316c3fafed32dfa639b951ddcfe28/figures/metrics.py#L678